submissions/BlessingOkoro/ecommerce-project/README - endpoints docs.md

```diff
@@ -0,0 +1,3 @@
+http://localhost:8888/welcome/                   #Home endpoint
```

Comment:
Good job adding this! Also add the expected http method for each endpoint, and sample requests